src/Anchu/Ftp/Ftp.php

```diff
@@ -30,11 +30,15 @@ public function connect($config)
     {
         if(!isset($config['port']))
             $config['port'] = 21;
-
-        $connectionId = ftp_connect($config['host'],$config['port']);
-        $loginResponse = ftp_login($connectionId, $config['username'], $config['password']);
-        ftp_pasv($connectionId, $config['passive']);
-        ftp_set_option($connectionId, FTP_TIMEOUT_SEC, 300);
+        if(!isset($config['timeout']))
+            $config['timeout'] = 30;
+
+        $connectionId = ftp_connect($config['host'],$config['port'],$config['timeout']);
+        if ($connectionId) {
+            $loginResponse = ftp_login($connectionId, $config['username'], $config['password']);
+            ftp_pasv($connectionId, $config['passive']);
+            ftp_set_option($connectionId, FTP_TIMEOUT_SEC, 300);
```

Comment:
We already set timeout value, hence I think this line is not needed. Can you remove this line?